recipes/spectate/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "spectate" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d4d985589d755b4bf118696eb794bc35c4131809be812eecd95fd9938809efbf
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.0
```

Comment:
can this package be `noarch`?

Reply:
I thought it was, but because it's crashing on windows it means it isn't right :p

Reply:
for a noarch package you cannot have version specifiers, and you need to add `noarch: python` in the build section (where you also specifiy the build number etc.)

Reply:
for noarch you can't have any version specifiers and you need to add `noarch: python` to the build section. e.g.

```
build:
  number: 0
  noarch: python
...
```

Reply:
this will generate 1 package that can be installed on any operating system and with any python version so it uses less space and CI time :)
This generally works for pure-python packages.

Reply:
You mean I can't have version specifiers in my yaml file?
Like I have `    - python >=3.0` for instance? 
This is a pure python one.

Reply:
exactly that's what i meant :)